pkg/netlink/fdb.go

```diff
@@ -0,0 +1,149 @@
+package netlink
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/opiproject/opi-evpn-bridge/pkg/infradb"
+)
+
+// ParseFdb parse the fdb
+func ParseFdb(fdbIP FdbIPStruct) *FdbEntryStruct {
```

Comment:
functions are usually kept in one file with their types. You can move types definitions closer to their functions